.freeCodeCamp/tooling/locales/english/build-a-peer-to-peer-network.md

```diff
@@ -174,7 +166,7 @@ assert.include(res3.data, 'ws://localhost:4003', `Your node on port 4003 should
 ### --before-all--
 
 ```js
-global.root = '..';
+global.root = '.';
 global.projectFolder = `${root}/build-a-peer-to-peer-network`;
```

Comment:
```suggestion
global.projectFolder = `build-a-peer-to-peer-network`;
```

Reply:
I prefer to leave this one for now. All my other projects still have it and it keeps it real easy to change if something changes again - although, I doubt it will.

Reply:
Sure. It was a nitpick anyway